app/src/main/java/com/eveningoutpost/dexdrip/ui/MicroStatusImpl.java

```diff
@@ -28,12 +31,20 @@ public String gs(String id) {
     }
 
     @Override
-    public boolean bluetooth() {
-        return DexCollectionType.hasBluetooth();
+    public boolean bluetooth() { // Dexcom with Bluetooth except G7
+        return DexCollectionType.hasBluetooth() && !using_g7();
```

Comment:
not sure if you are braking logic for other sensors with the extra check

Reply:
I don't know what you mean by that.  Please explain.

Reply:
I see what you mean now.  
First, let me thank you for spending the time to review.

Please let me explain this concern.
There are only two possibilities under all circumstances.  Either we are using G7 or not.

If we are not using G7, what is being anded is true.  Anding true with anything changes nothing.  Therefore, nothing changes with respect to other sensors.

If we are using G7, the returned value becomes false, which removes three lines from the status page.

Reply:
Wow!  You have a point.  There is a third case of the value being unknown.  I need to make sure if the value is unknown, it doesn't break anything.  Thanks so much.

Reply:
@DiBochev Is it OK now?